Client/src/Pages/PageSpeed/CreatePageSpeed/index.jsx

```diff
@@ -1,114 +1,73 @@
+// React, Redux, Router
+import { useNavigate, useParams } from "react-router-dom";
+import { useEffect } from "react";
 import { useState } from "react";
-import { Box, Button, ButtonGroup, Stack, Typography } from "@mui/material";
-import LoadingButton from "@mui/lab/LoadingButton";
 import { useSelector, useDispatch } from "react-redux";
+
+// Utility and Network
+import { logger } from "../../../Utils/Logger";
 import { monitorValidation } from "../../../Validation/validation";
-import { useNavigate } from "react-router-dom";
-import { useTheme } from "@emotion/react";
 import {
 	createPageSpeed,
 	checkEndpointResolution,
 } from "../../../Features/PageSpeedMonitor/pageSpeedMonitorSlice";
-import { createToast } from "../../../Utils/toastUtils";
-import { logger } from "../../../Utils/Logger";
-import { ConfigBox } from "../../Monitors/styled";
-import Radio from "../../../Components/Inputs/Radio";
+import { parseDomainName } from "../../../Utils/monitorUtils";
+
+// MUI
+import { useTheme } from "@emotion/react";
+import { Box, Stack, Typography, Button, ButtonGroup } from "@mui/material";
+import LoadingButton from "@mui/lab/LoadingButton";
+
+//Components
+import Breadcrumbs from "../../../Components/Breadcrumbs";
 import TextInput from "../../../Components/Inputs/TextInput";
 import { HttpAdornment } from "../../../Components/Inputs/TextInput/Adornments";
-import Select from "../../../Components/Inputs/Select";
+import { ConfigBox } from "../../Monitors/styled";
+
+import { createToast } from "../../../Utils/toastUtils";
+import Radio from "../../../Components/Inputs/Radio";
 import Checkbox from "../../../Components/Inputs/Checkbox";
-import Breadcrumbs from "../../../Components/Breadcrumbs";
-import { parseDomainName } from "../../../Utils/monitorUtils";
-import "./index.css";
+import Select from "../../../Components/Inputs/Select";
 
 const CreatePageSpeed = () => {
 	const MS_PER_MINUTE = 60000;
-	const { user, authToken } = useSelector((state) => state.auth);
-	const { isLoading } = useSelector((state) => state.pageSpeedMonitors);
-	const dispatch = useDispatch();
-	const navigate = useNavigate();
-	const theme = useTheme();
 
-	const idMap = {
-		"monitor-url": "url",
-		"monitor-name": "name",
-		"monitor-checks-http": "type",
-		"monitor-checks-ping": "type",
-		"notify-email-default": "notification-email",
-	};
+	const CRUMBS = [
+		{ name: "pagespeed", path: "/pagespeed" },
+		{ name: "create", path: `/pagespeed/create` },
+	];
 
+	const SELECT_VALUES = [
+		{ _id: 3, name: "3 minutes" },
+		{ _id: 5, name: "5 minutes" },
+		{ _id: 10, name: "10 minutes" },
+		{ _id: 20, name: "20 minutes" },
+		{ _id: 60, name: "1 hour" },
+		{ _id: 1440, name: "1 day" },
+		{ _id: 10080, name: "1 week" },
+	];
```

Comment:
I really like the constant approach. I suggest we put them out of the component (line 31) so they don't get recreated on rerender (since they don't depend on props. If they did, it would be a case for using useMemo)

Reply:
Will do 👌